app/templates/partials/questions/relationship.html

```diff
@@ -1,26 +1,28 @@
-{% import 'macros/helpers.html' as helpers %}
-
-<div class="relationship">
-
-    <h2 class="question__title neptune">
+<div class="question question--repeatinganswer" id="{{question.id}}">
+
+  <h2 class="question__title neptune">
+    {%- if question_number -%}
+      <span class="question__number">{{question_number}}{{current_location.group_instance|format_number_to_alphabetic_letter}}. </span>
+    {% endif %}
+    {{question_title|safe}}
+  </h2>
+
+  {{question_description|safe}}
+  <fieldset>
+    <legend class="field__legend mars u-vh">
       {%- if question_number -%}
-        <span class="question__number">{{question_number}}. </span>
+        <span class="question__number">{{question_number}}{{current_location.group_instance|format_number_to_alphabetic_letter}}. </span>
```

Comment:
I would remove the number from the `legend` to be consistent with other templates.